python/lsst/ts/analysis/notebooks/analysisTools.py

```diff
@@ -0,0 +1,399 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.visualization import ZScaleInterval
+from lsst.daf import butler as dafButler
+
+from matplotlib import rcParams
+
+rcParams["ytick.labelsize"] = 15
+rcParams["xtick.labelsize"] = 15
+rcParams["axes.labelsize"] = 20
+rcParams["axes.linewidth"] = 2
+rcParams["font.size"] = 15
+rcParams["axes.titlesize"] = 18
+
+# common functions for AOS analysis
+
+
+def plotZernikeImageFromPickle(
+    expIntra,
+    yearMonthDay,
+    zkResultsFile=None,
+    collection="u/scichris/Latiss/postISRex",
+    repoDir="/repo/main/",
+):
+    """Plot the raw Zernikes and postISR image from pickle file.
+
+    Parameters
+    ----------
+    expIntra: int
+        Exposure number, eg. 487
+    yearMonthDay: str
+        A string defining year, month, and day of observations,
+        eg. "20210908"
+    zkResultsFile: str, optional
+        Name of the pickle file with Zernike results
+        (by default it is None, and the name of the file
+        is constructed from expIntra, yearMonthDay provided).
+    collection: str, optional
+        Collection name containing the postISRCCD. By
+        default is is u/scichris/Latiss/postISRex
+    repoDir : str, optional
+        Path to a directory containing butler.yaml file.
+        For Latiss tests default is /repo/main
+    """
+    # plot the figure ...
+    fig = plt.figure(figsize=(14, 5))
+
+    ################################
+    # left - plot the fit results  #
```

Comment:
I wonder if you could break the left/right plots into individual plotting functions... .